apps/backoffice-v2/src/pages/TransactionMonitoringAlerts/components/AlertsTable/columns.tsx

```diff
@@ -44,6 +44,14 @@ export const columns = [
     },
     header: 'Date & Time',
   }),
+  columnHelper.accessor('label', {
+    cell: info => (
+      <Badge variant="secondary" className="max-w-[8rem]" title={info.getValue()}>
+        <div className="truncate">{info.getValue()}</div>
+      </Badge>
+    ),
```

Comment:
```suggestion
    cell: info => {
    const label = info.getValue();
    
    return (
      <Badge variant="secondary" className="max-w-[8rem]" title={label}>
        <div className="truncate">{label}</div>
      </Badge>
    )
    },
```

Reply:
When the `truncate` is on the `Badge` component it doesn't look good, it ignores the padding, etc.